lib/suspenders/app_builder.rb

```diff
@@ -98,7 +102,9 @@ def enable_rack_canonical_host
       config = <<-RUBY
 
   # Ensure requests are only served from one, canonical host name
-  config.middleware.use Rack::CanonicalHost, ENV.fetch("APPLICATION_HOST")
+  unless ENV["HEROKU_APP_NAME"].include?("staging-pr-")
```

Comment:
Technically `staging-` would reach the same effect. The staging app doesn't have a dash suffix or anything after.